### DISCUSSION TITLE
How to give full play to the visual processing power of paraview？

### DISCUSSION THREAD

**Discussion by Xiaozizhuo** [PMEAL/OpenPNM#2402] [Q&A]
Dear Professor Jeff, is there a more detailed paraview tutorial on post-processing of porous media models?

For example, in the following figure, the size of the pores and the distribution of pressure in the pores are perfectly displayed, but the size distribution of the larynx does not seem to be obvious, how to make the distribution of the larynx more vivid?
![openpnm](https://user-images.githubusercontent.com/98071787/166440027-4383f555-0ab9-4672-8264-d6c02988d618.png)

**Comment by jgostick** [comment]
visualizing the throats is a bit of a pain but is do-able.  There is a [tutorial here](https://github.com/PMEAL/OpenPNM/blob/dev/examples/getting_started/view_vtp_in_paraview.md).  It's hard to find at the moment.
